### ISSUE TITLE
Is this package abandoned in favor of GraphMakie?

### ISSUE THREAD

**Issue by abelsiqueira** [JuliaGraphs/GraphPlot.jl#198]
Is this package abandoned in favor of GraphMakie?

**Comment by hdavid16** [comment]
As far as I know, it isn't being actively maintained. GraphMakie is and has good support.

**Comment by abelsiqueira** [comment]
We are making the move, thanks!

**Comment by hdavid16** [comment]
It's unfortunate this package is not being maintained. I find it easier to use for quick plots, but GraphMakie gives you more customization.

**Comment by abelsiqueira** [comment]
Indeed, it was a quick solution, but I actually came here to create some issues about some arguments not working as expected/desired. Using GraphMakie solved the problems.

**Comment by hdavid16** [comment]
On a side note, they still list GraphPlot under the packages section: https://juliagraphs.org/packages/

**Comment by hyrodium** [comment]
GraphPlot.jl is also listed in https://juliagraphs.org/Graphs.jl/dev/first_steps/plotting/.
